README.md

```diff
@@ -1,10 +1,31 @@
+# stack
```

Comment:
Right at the head of MinGHC seems a bit surprising. Perhaps this could be a subsection instead?